app/js/enhancedDate.js

```diff
@@ -1,5 +1,76 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function(window, undefined) {
 
+	var _date;
+	var weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+	var month = ["January", "February", "March", "April", "May", "June", "July", "Auguest", "September", "October", "Novenber", "December"];
+	var current = new Date();
```

Comment:
Setting the default date here is premature. As soon as this code executes, which is as soon as the application that uses it starts, the date will be set. If the application runs for a long time, this date becomes out-of-date. Better to set it when it's needed.